linode/manager
# refactor: [M3-7542] - TagsInput & TagsPanel Storybook v7 Stories

## Description 📝
Small PR to handle `.tsx` v7 stories for `TagsInput` & `TagsPanel`

There is no change that should affect client-side code. This only touches our book.

## Changes  🔄
- New v7 stories for `TagsInput` & `TagsPanel`
- New test for `TagsPanel`
- Move styles in own file (`TagsPanel`)
- Remove stories list within documentation

## Preview 📷
![Screenshot 2023-12-06 at 11 42 58](https://github.com/linode/manager/assets/130582365/38d0c203-f26e-4c8e-bc63-eb839a7d0227)

![Screenshot 2023-12-06 at 11 42 58](https://github.com/linode/manager/assets/130582365/4c67a6d6-9248-4f2d-bf4a-95b73e181559)

## How to test 🧪

### Verification steps 
- pull code and run `yarn storybook`
- verify http://localhost:6006/?path=/docs/components-tags-tags-input--documentation
- verify http://localhost:6006/?path=/docs/components-tags-tags-panel--documentation
- run `yarn test TagsPanel.test.tsx`

## As an Author I have considered 🤔

*Check all that apply*
